src/lib/y2network/clients/routing_auto.rb

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "installation/auto_client"
+require "y2network/config"
+require "y2network/routing"
+require "y2network/autoinst_profile/networking_section"
+require "y2network/config_reader/autoyast"
+
+Yast.import "NetworkInterfaces"
+Yast.import "Lan"
+
+module Y2Network
+  module Clients
+    # This client is reponsible of the autoinstallation routing configuration.
```

Comment:
It will be completely dropped

Reply:
Then I would say that it does not make sense to merge this.